tests/kani/Makefile

```diff
@@ -0,0 +1,38 @@
+SHELL=/bin/bash
+
+# Assume cbmc and cbmc-viewer are installed in PATH
+#   skip ./kani/scripts/setup/ubuntu/install_cbmc.sh
+#   skip ./kani/scripts/setup/install_viewer.sh 2.11
+
+KANI=kani
+
+# ARCH is ubuntu or macos-10.15
+ARCH=ubuntu
```

Comment:
Is this expected to be modified by the user? If so, shouldn't we use `?=` so that the environment variable overrides it if set?

Reply:
Done.  I have been setting it from the make command.